src/util/configInheritance.tsx

```diff
@@ -118,6 +121,11 @@ const getNetworkRowMetadata = (name: string): ConfigRowMetadata => {
   return getNetworkDefault(name);
 };
 
+// NOTE: this is only relevant for Ceph RBD storage pools at the moment
+const getStoragePoolRowMetadata = (name: string): ConfigRowMetadata => {
+  return getCephStoragePoolDefault(name);
```

Comment:
This is pretty neat for the follow-up, when the defaults come in via API, which is planned to be added soon.